style.css

```diff
@@ -24,6 +24,152 @@ html {
   color: #1a2236;
 }
 
+/* -----------------------------POPUP CSS SECTION STARTS------------------------------------ */
+
+.popup__container {
+  width: 100%;
+  padding: 16px;
+  gap: 25px;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  background-color: #fff;
+  border: #1a2236 28px solid;
+}
+
+.popup__title {
+  align-self: flex-start;
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 20px;
+  line-height: 80%;
+  color: #1a2236;
+}
+
+.popup__img {
+  width: 100%;
+}
+
+.popup__feature {
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 18px;
+  line-height: 27px;
+  color: #1a2236;
+}
+
+.btn6 {
+  width: 100%;
+  padding: 0;
+  margin: 0;
+  display: flex;
+  justify-content: center;
+  flex-direction: column;
+  align-items: center;
+  gap: 50px;
+}
+
+.popup__btn {
+  width: 80%;
+  padding: 10px 20px;
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 16px;
+  line-height: 24px;
+  text-align: center;
+  background: #fff;
+  color: #1a2236;
+  transition: 0.5s;
```

Comment:
[OPTIONAL]
- [ ] Kindly you may consider reducing the spacing between your button as you may notice how large they are.
- [ ] The same applies to the size of your close button. :smile: 
Your design :point_down: 
![image](https://user-images.githubusercontent.com/97100091/202838086-0018ad91-d917-4213-8a85-7d453d7cf34b.png)

![image](https://user-images.githubusercontent.com/97100091/202838132-54149737-48bd-44f7-90a4-f15b5c57bd93.png)